mlflow/lightgbm.py

```diff
@@ -297,6 +346,52 @@ def log_model(
     :param kwargs: kwargs to pass to `lightgbm.Booster.save_model`_ method.
     :return: A :py:class:`ModelInfo <mlflow.models.model.ModelInfo>` instance that contains the
              metadata of the logged model.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+        from lightgbm import LGBMRegressor
+        import numpy as np
+
+
+        def gen_data():
+            # Dummy data for f(x) = 2x
+            # X training data, y labels
+            X = np.arange(1, 51)
+            y = 2 * X
+            return X.reshape(-1, 1), y
```

Comment:
Can we use the iris dataset to shorten the example code?

Reply:
Sure, I will make the necessary changes.

Reply:
@canerturkseven Can you also fix the DCO check failure!

Reply:
Hi @harupy, I made the proposed changes and fixed the DCO failure.